kotlin/choose-camera-controller/README.md

```diff
@@ -26,9 +26,6 @@ The application loads with the default globe camera controller. To rotate and fi
 *   OrbitLocationCameraController
 *   SceneView
 
```

Comment:
⬆️ the How it works and Relevant API sections have extra whitespace between the numbers/asterisks and words. Also, the How it works section needs to fix the numbering.